src/NuGet.Core/NuGet.Indexing/NuGet.Indexing.csproj

```diff
@@ -4,6 +4,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
+    <!-- TODO NK Move to clients, this is dumb-->
```

Comment:
Does it need a task/workitem link?

Reply:
Yeah, it does :D 
I usually scan for those before PRing but apparently I missed it. 

Created a public bug as it would affect anyone who has our repo checked out, https://github.com/NuGet/Home/issues/9245